convert/convert_test.go

```diff
@@ -96,6 +96,7 @@ func TestConvertModel(t *testing.T) {
 		"Mistral-7B-Instruct-v0.2",
 		"Mixtral-8x7B-Instruct-v0.1",
 		"gemma-2b-it",
+		"gemma-2-2b-it",
```

Comment:
there's already a 9b so this isn't necessary

Reply:
The 9b model test is missing a lot of stuff in it and didn't catch the bug, which is why I added this one.

Reply:
I've added a check for the duplicate tensors as well as a unit test so we can see duplicate tensors more quickly.

For 9b vs 2b you can skip the test by not including a link to the safetensors model; given that they're slightly different (and that we don't have the full test for 9b) I think it's worth having the test.

Reply:
imo it would better to fill in the missing tensors than to add a new test case. there's significant overlap between 2b and 9b when it comes to the conversion process so adding a new test case is unnecessary

alternatively you can remove the 9b test case